rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy

```diff
@@ -606,6 +654,36 @@ class ProjectController extends ControllerBase{
     /**
      * API: /api/11/project/NAME
      */
+    @Get('/{project}')
+    @Operation(
+            method = "GET",
+            summary = "Get a project",
+            description = """Get information about a project.
+The reponse in XML or JSON format is determined by the Accept request header.
+
+Authorization required: `read` access for `project` resource type to get basic project details and `configure` access to get all properties config or `admin` or `app_admin` access for `user` resource type.""",
+            parameters = @Parameter(
+                    name = 'project',
+                    in = ParameterIn.PATH,
+                    description = 'Project Name',
+                    allowEmptyValue = false,
+                    required = true,
+                    schema = @Schema(implementation = String.class)
+            )
+    )
+    @Tags(
+            [
+                    @Tag(name="project")
+            ]
+    )
+    @ApiResponse(
+            responseCode = "200",
+            description = "Project details",
+            content = @Content(
+                    mediaType = "application/json",
```

Comment:
I checked the result file using `autorest` (what we used for generate the `go-rundeck` client) and complained the `schema` is missing here